spec/fbp.coffee

```diff
@@ -136,6 +226,28 @@ describe 'FBP parser', ->
         chai.expect(graphData.inports).to.be.an 'undefined'
         chai.expect(graphData.outports).to.be.an 'undefined'
 
+  describe 'with FBP string containing a JSON IIP string', ->
```

Comment:
Can you also add a test for couple leves of nested object? It is the kind of thing which can easily break, in my experience